src/index.js

```diff
@@ -756,23 +756,66 @@ export class ULabel {
         });
     }
 
+    static handle_deprecated_arguments() {
+        return {
+            "container_id": arguments[0],  // Required
+            "image_data": arguments[1],    // Required
+            "username": arguments[2],      // Required
+            "submit_butons": arguments[3], // Required
+            "subtasks": arguments[4],      // Required
```

Comment:
Could add an error message here for when fewer than five args are passed in letting the user know that the require some

Reply:
could also just add a warning in general to indicate that this argument style is deprecated

Reply:
Adding a warning that the old method is deprecated is a good idea. However the error message is already implemented on lines 788-800. It does this be delaying an arrray of required properties and looping through it to check to make sure that kwargs contains all properties. If at least one is missing it will log an error stating which property is missing.